packages/graphql-server/src/resolvers/contact/__mocks__/create-args.ts

```diff
@@ -1,48 +1,51 @@
-import { CreateContactArgs } from "../contact";
+import { CreateContactArgs } from '../contact';
 
 export const mockCreateArgs = {
-  id: 'MKC16000098',
-  created: '2019-04-23T16:05:19.0000000Z',
-  modified: null,
-  title: 'Ms',
-  forename: 'Saoirse',
-  surname: 'Chadwick',
-  dateOfBirth: '1986-06-30',
+  title: 'Mr',
```

Comment:
Again, should be typed

Reply:
Hi @willmcvay its already been typed as CreateContactArgs